.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,15 +8,49 @@
 
 ## Summary ##
 
-<!-- Describe the change, including rationale and design decisions -->
+<!--
+Describe the change, including rationale and design decisions, and use a brief
+but descriptive PR title.
+
+The title should be prefixed with `[WIP]` before the checklist below is
```

Comment:
Could also be handled by labels I suppose. If the person is part of the project then s/he can apply the label, if not, then opening a PR from a forked repository means that a review is needed - so it is implied that work on a remote branch without PR is a WIP. WDYS?